server/utils.go

```diff
@@ -196,3 +198,15 @@ func recordError(operation string, err error) {
 		metrics.CRIOOperationsErrors.WithLabelValues(operation).Inc()
 	}
 }
+
+func validateLabels(labels map[string]string) error {
+	for k, v := range labels {
+		if (len(k) + len(v)) > maxLabelSize {
+			if len(k) > 10 {
+				k = k[:10]
```

Comment:
This makes me nervous, modifying the value in the loop, since you are returning out of it, it is probably not an issue. but if we added a new variable `key` and printed that, it would make me less nervous.

Reply:
If we do this, we always return an error so there's no harm in doing that here afaict